api/src/main/java/org/openmrs/module/aijar/activator/AlertConfigurationInitializer.java

```diff
@@ -0,0 +1,42 @@
+package org.openmrs.module.aijar.activator;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.module.aijar.api.AijarService;
+
+import javax.naming.Context;
+
+import static org.openmrs.module.aijar.AijarConstants.*;
+import static org.openmrs.module.aijar.AijarConstants.GP_DHIS2_DEFAULT_ALERT_MESSAGE;
+import static org.openmrs.module.aijar.AijarConstants.GP_DHIS2_VALUE;
+
+public class AlertConfigurationInitializer implements Initializer{
+    protected Log log = LogFactory.getLog(AlertConfigurationInitializer.class);
+    @Override
+    public void started() {
+        AdministrationService administrationService= org.openmrs.api.context.Context.getAdministrationService();
+        AijarService aijarService= org.openmrs.api.context.Context.getService(AijarService.class);
+        // set alert if National Health Provider Identifier is not set
+        if (administrationService.getGlobalProperty(GP_NHPI).equalsIgnoreCase(GP_NHPI_VALUE)) {
```

Comment:
@slubwama Instead of the 3 If statements create a private method that sends the alerts then call that instead